config/locales/en/views.yml

```diff
@@ -195,8 +195,11 @@ en:
         in the box below.
       feedback_form: Feedback form
       body: Your message
-      body_hint: >-
-        Do not include sensitive personal information, such as prisoner numbers.
+      date_of_birth: Prisoner date of birth
```

Comment:
Tricky because the view code that calls this is a helper (`composite_field`) which derives the i18n scope implicitly based on the model calling it. After discussing with @alan it was decided to maybe raise a ticket to de-duplicate (where possible) entries in the locale .yml files.